<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TalesQtPowerUpEditorWidget</class>
 <widget class="QWidget" name="TalesQtPowerUpEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>129</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="EmeraldEditorWidget" name="emeraldEditorWidget_0" native="true"/>
        </item>
        <item>
         <widget class="EmeraldEditorWidget" name="emeraldEditorWidget_1" native="true"/>
        </item>
        <item>
         <widget class="EmeraldEditorWidget" name="emeraldEditorWidget_2" native="true"/>
        </item>
        <item>
         <widget class="EmeraldEditorWidget" name="emeraldEditorWidget_3" native="true"/>
        </item>
        <item>
         <widget class="EmeraldEditorWidget" name="emeraldEditorWidget_4" native="true"/>
        </item>
        <item>
         <widget class="EmeraldEditorWidget" name="emeraldEditorWidget_5" native="true"/>
        </item>
        <item>
         <widget class="EmeraldEditorWidget" name="emeraldEditorWidget_6" native="true"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>129</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EmeraldEditorWidget</class>
   <extends>QWidget</extends>
   <header>emeraldeditorwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
